app/scripts/views/base.js

```diff
@@ -888,15 +888,13 @@ define(function (require, exports, module) {
       }
 
       if (_.isFunction(handler)) {
-        var args = [].slice.call(arguments, 1);
-
         // If an `arguments` type object was passed in as the first item,
         // then use that as the arguments list. Otherwise, use all arguments.
         if (_.isArguments(args[0])) {
           args = args[0];
         }
 
-        return handler.apply(this, args);
+        return handler.call(this, ...args);
```

Comment:
> Better to continue leaning on the JS interpreter to apply the arguments for us, than whatever Babel transpiles the spread operator to. Am I being unreasonable?

It sounds like there are two concerns:

1. The change from `apply` to `call` is more or less pointless because the end result is the same.
2. Lack of trust in Babel.

The first argument I buy, the second not so much. If we don't trust Babel to do the right thing, we shouldn't be using it.

I'll change it back because I agree with the first and when making the update wondered the same thing.